examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,135 @@
+"""
+=======================================================================
+Compute MNE inverse solution on evoked data in a mixed source space
+=======================================================================
+
+Create a mixed source space and compute MNE inverse solution on evoked dataset
+
+"""
+# Author: Annalisa Pascarella <a.pascarella@iac.cnr.it>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import matplotlib.pyplot as plt
+import mne
+
+from mne.datasets import sample
+from mne import write_source_spaces, setup_source_space
+from mne import setup_volume_source_space
+from mne import make_forward_solution
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+
+from nilearn import plotting
+
+# Set dir
+data_path = sample.data_path()
+subject = 'sample'
+data_dir = op.join(data_path, 'MEG', subject)
+subjects_dir = op.join(data_path, 'subjects')
+bem_dir = op.join(subjects_dir, subject, 'bem')
+
+# Set file names
+fname_mixed_src = op.join(bem_dir, '%s-oct-6-mixed-src.fif' % subject)
+fname_aseg = op.join(subjects_dir, subject, 'mri', 'aseg.mgz')
+
+fname_model = op.join(bem_dir, '%s-5120-bem.fif' % subject)
+fname_bem = op.join(bem_dir, '%s-5120-bem-sol.fif' % subject)
+
+fname_evoked = data_dir + '/sample_audvis-ave.fif'
+fname_trans = data_dir + '/sample_audvis_raw-trans.fif'
+fname_fwd = data_dir + '/sample_audvis-meg-oct-6-mixed-fwd.fif'
+fname_cov = data_dir + '/sample_audvis-shrunk-cov.fif'
+
+# List of sub structures we are interested in. We select only the
+# sub structures we want to include in the source space
+labels_vol = ['Left-Amygdala',
+              'Left-Thalamus-Proper',
+              'Left-Cerebellum-Cortex',
+              'Brain-Stem',
+              'Right-Amygdala',
+              'Right-Thalamus-Proper',
+              'Right-Cerebellum-Cortex']
+
+# Setup a surface-based source space
+src = setup_source_space(subject, subjects_dir=subjects_dir,
+                         spacing='oct6', add_dist=False, overwrite=True)
```

Comment:
@agramfort if I remove `overwrite=True` in `setup_source_space` I get the exception 
`IOError: file "/home/pasca/mne_data/MNE-sample-data/subjects/sample/bem/sample-oct-6-src.fif" exists, use overwrite=True if you want to overwrite the file`

what is the best way to proceed? thks!